### PR TITLE
expand collapse card changes

### PR DESCRIPTION
**Pull request by Suyashi123**

### CODE REVIEW

File: src/Pages/License/Search/index.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Content from "Layouts/Content/Content";
 
 import SearchLicenseDetails from "./Details/SearchLicenseDetails";
-import SearchLicense from "./SearchLicense";
+import SearchLicense from "./searchLicense";
```
Comment (pinak-globallogic) [inline src/Pages/License/Search/index.jsx:4]: Fix

**Review by pinak-globallogic — CHANGES_REQUESTED** [review]

File: src/Pages/License/Generate/GenerateLicenseWizard.jsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import { useState } from "react";
+//import { useState } from "react";
```
Comment (pinak-globallogic) [inline src/Pages/License/Generate/GenerateLicenseWizard.jsx:1]: remove

File: src/Pages/License/Generate/GenerateLicenseWizard.jsx [hunk 3]
```diff
@@ -93,7 +102,13 @@ const GenerateLicenseWizard = (props) => {
               size="small"
               disabled={btnDisabled}
               onClick={() => {
-                setCount(count - 1);
+               // setCount(count - 1);
```
Comment (pinak-globallogic) [inline src/Pages/License/Generate/GenerateLicenseWizard.jsx:105]: remove

File: src/Pages/License/Generate/GenerateLicenseWizard.jsx [hunk 4]
```diff
@@ -23,18 +24,26 @@ const noOfKeys = (keyAmount) => {
...
+  
   const btnDisabled = state.error;
 
   const renderSwitch = (count) => {
     let obj = {
       btn: {
         text: "continue",
         variant: "contained",
-        action: () => setCount(count + 1),
+        action: () => //setCount(count + 1),
```
Comment (pinak-globallogic) [inline src/Pages/License/Generate/GenerateLicenseWizard.jsx:40]: remove all comments

**Review by pinak-globallogic — CHANGES_REQUESTED** [review]

File: src/Pages/License/Search/index.jsx (hunk 1 above)
Comment (Suyashi123) [reply src/Pages/License/Search/index.jsx]: done

File: src/Pages/License/Generate/GenerateLicenseWizard.jsx (hunk 2 above)
Comment (Suyashi123) [reply src/Pages/License/Generate/GenerateLicenseWizard.jsx]: done

File: src/Pages/License/Generate/GenerateLicenseWizard.jsx (hunk 4 above)
Comment (Suyashi123) [reply src/Pages/License/Generate/GenerateLicenseWizard.jsx]: done

File: src/Pages/License/Generate/GenerateLicenseWizard.jsx (hunk 3 above)
Comment (Suyashi123) [reply src/Pages/License/Generate/GenerateLicenseWizard.jsx]: done

**Review by pinak-globallogic — APPROVED** [review]

**Pull request merged**